media/admin/com_virtuemart/views/zasilkovna/tmpl/order_detail_form.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Template variables
+ *
+ * @return mixed
+ * @var \VirtueMartModelZasilkovna\Order\ShipmentInfo $shipment
+ */
+
+defined('_JEXEC') or die;
+
+$adultContentChecked = $shipment->getAdultContent() ? 'checked' : '';
+$token = \JHtml::_('form.token');
+
+?>
+<a id="showPacketeryUpdateOrderDetail" class="btn btn-small" href="#"><?php echo(JText::_('PLG_VMSHIPMENT_PACKETERY_EDIT_PACKET_DETAIL')); ?>
+    <span class="vmicon vmicon-16-editadd"></span>
+</a>
+<div id="packeteryUpdateOrderDetail" class="vm-absolute">
+        <form action="index.php?option=com_virtuemart&view=zasilkovna&task=updatePacketeryOrderDetail" method="post" id="packeteryUpdateOrderDetailForm">
+            <div>
+                <input type="hidden" name="virtuemart_order_id" value="<?php echo($shipment->getVirtuemartOrderId()); ?>">
+                <input type="hidden" name="order_number" value="<?php echo($shipment->getOrderNumber()); ?>">
+                <?php echo $token; ?>
+                <fieldset>
+                    <table class="admintable table">
+                        <thead>
+                            <tr>
+                                <td colspan="2">
+                                    <h1><?php echo(JText::_('PLG_VMSHIPMENT_PACKETERY_EDIT_DETAIL')); ?></h1>
+                                </td>
+                            </tr>
+                        </thead>
+                        <tr>
+                            <td class="key va-middle"><?php echo(JText::_('PLG_VMSHIPMENT_PACKETERY_WEIGHT')); ?></td>
+                            <td ><input type="number" step="0.1" name="weight" value="<?php echo($shipment->getWeight()); ?>"> kg</td>
+                        </tr>
+                        <tr>
+                            <td class="key va-middle"><?php echo(JText::_('PLG_VMSHIPMENT_PACKETERY_COD')); ?></td>
+                            <td>
+                                <input type="number" step="0.01" name="packet_cod" value="<?php echo($shipment->getPacketCod()); ?>">
+                                <?php echo($shipment->getBranchCurrency()); ?>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td class="key va-middle"><?php echo(JText::_('PLG_VMSHIPMENT_PACKETERY_PACKET_PRICE')); ?></td>
+                            <td>
+                                <input type="number" step="0.01" name="zasilkovna_packet_price" value="<?php echo($shipment->getZasilkovnaPacketPrice()); ?>">
+                                <?php echo($shipment->getBranchCurrency()); ?>
```

Comment:
zde bych použil htmlentities(). Já bych to dal všude. I tam kde se teď tisknout jen čísla.